NEWS.md

```diff
@@ -1,7 +1,9 @@
-# palaeoverse 1.1.0.9...
+# palaeoverse 1.1.1
 
 * palaeoverse now requires deeptime (>= 1.0.0)
 * Added additional test to catch when Zenodo is offline (palaeorotate)
+* palaeorotate documentation updated and MULLER2022 model made available
+* uncertainty argument in palaeorotate updated to enable user-input
```

Comment:
Make sure to update the news accordingly with whatever changes are made based on my comments!
(and you should probably cite the issue that you are fixing)